src/components/Navigation/Navigation.jsx

```diff
@@ -37,32 +32,26 @@ export default function Navigation(){
 		}
 	];
 
-
-	const links       = linkData.map(RENDER.item);
-	const ariaOpen    = open.toString();
+	const links      	  = linkData.map(RENDER.item);
+	const ariaExpanded    = expanded.toString();
 
 	return (
-		<nav>
-			<button 
-				role="switch"
-				aria-checked={ ariaOpen }
-				aria-controls="navigation__links"
-				onClick={ toggleOpen }
-			>
-				<span>
-					Menu
-				</span>
-				<span>
-					Close
-				</span>
-			</button>
+		<nav className={ s.wrapper }>
+			<NavProvider>
```

Comment:
if we take the provider out here and wrap this whole `Navigation` component within it (ie. in `<App />`, then we can use its `open` state instead of all the `expanded` stuff in here.

Reply:
Cool, wasn't sure the best way to do this :)